packages/devtools_app/lib/src/table.dart

```diff
@@ -673,6 +689,34 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
     sortColumn = widget.sortColumn;
     sortDirection = widget.sortDirection;
     scrollController = ScrollController();
+
+    if (widget.activeSearchMatchNotifier != null) {
+      widget.activeSearchMatchNotifier
+          .addListener(activeSearchListener = _onActiveSearchChange);
+
+      // TODO(kenz): why isn't the listener firing when added via
```

Comment:
that should be equivalent. Maybe you needed a hot restart for the change to have an effect.
This sort of change should be duplicated in didUpdateWidget as well as initState or there will be issues. Seems like a lot of the logic in the table class has that same issue.